src/components/styles/Navbar.module.css

```diff
@@ -0,0 +1,69 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 11%;
+  font-family: 'Montserrat', sans-serif;
+  box-shadow: 1px 0 3px rgb(165, 164, 164);
+}
+
+.nav > div > ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  width: 170px;
+}
+
+.nav > div:nth-child(1) {
+  display: flex;
+  align-items: center;
+}
```

Comment:
- [ ] I would suggest you to stick to the ziplin design as much as posible to make your app look professional ,kindly make the TExt Bookstor align with lin bpx as show on the design template . 
- [ ] Your design
![Screenshot 2023-09-28 191307](https://github.com/nilab-khwaja/bookstore/assets/92458236/df6a22e9-7b6a-4392-b07a-589b23422546)

- [ ] Given design
![bookstore](https://github.com/nilab-khwaja/bookstore/assets/92458236/4bce56df-5c6c-4e1c-b18e-51d2d95a4f04)